src/Components/Selector/Selector.js

```diff
@@ -49,15 +49,15 @@ const Selector = ({getAndSetLocationData}) => {
   return (
     <form className='form' >
       <label> Select State:
-          <select onChange={event => setState(event.target.value)}>
-            <option value=''>None</option>
-            {eachState()}
-          </select>
-        </label>
-      <label> Select City: 
+        <select onChange={event => setState(event.target.value)}>
+          <option value=''>None</option>
+          {eachState()}
+        </select>
+      </label>
+      <label> Select City:
         <select onChange={event => setCity(event.target.value)}>
           <option value=''>None</option>
-          {availableCities.length && eachCity()}
+          {availableCities.length && eachCity() || <option value='select a state'>Select a State</option>}
```

Comment:
I love this solution! But I get an error when I click on the option on the right side of the `||` statement. I think we need to change that value statement to `value=''`. Otherwise it recognizes a city called `select a state` and does an API call we don't want. We could also fix the issue by moving that `||` logic to line 59. It would just be changing the text that appears there so something like: 
```
<option value=''>{state? 'None' : 'Select a State'}</option>
```
I think the second option would be slightly cleaner, but both work just fine. It's probably not cleaner, it's just more my coding style. I'll post a screenshot of the error so you can see if you're getting it too.

Reply:
I have gone in and fixed the error message.